apps/system/lockscreen/js/lockscreen_state_manager.js

```diff
@@ -453,6 +487,11 @@
         // @see: SecureAppManager
         this.onSecureAppClosing();
         break;
+      case 'keyup':
+        if(this.lockScreenStates.passcodeEnabled && (evt.keyCode >= KEY_CODE_ZERO && evt.keyCode <= KEY_CODE_NINE)) {
```

Comment:
What will happen if passcode lock disabled? How user expected to dismiss screen lock in such case?